qa/suites/rgw/verify/tasks/ragweed.yaml

```diff
@@ -0,0 +1,6 @@
+tasks:
+- ragweed:
+    client.0:
+      force-branch: ceph-luminous
+      rgw_server: client.0
+      stages: prepare,check
```

Comment:
oops - the ragweed task doesn't appear to exist in luminous. can you please remove this ragweed.yaml file?